config-overrides.js

```diff
@@ -9,5 +10,9 @@ module.exports = override(
   fixBabelImports("@material-ui/icons", {
     libraryDirectory: "esm",
     camel2DashComponentName: false
+  }),
+
+  addWebpackAlias({
+    "@src": path.resolve(__dirname, "./src/")
```

Comment:
To avoid using relative paths with `../../something`. It's difficult to reason about and it's more likely to lead to problems in the future